styles/components/alert.scss

```diff
@@ -1,3 +1,40 @@
 .alert {
+	padding: 20px 40px;
+	margin: 10px;
```

Comment:
No margins! To put margins on the examples, add `doc-bottom-space` or `doc-bottom-space-large`.